man/memkind.3

```diff
@@ -524,6 +530,32 @@ section if it is not.
 .BR MEMKIND_PMEM_MIN_SIZE
 The minimum size which allows to limit the file-backed memory partition.
 .sp
+.B "STATISTICS:"
+.br
+The functions described in this section define a way to get specific memory allocation statistics.
+.PP
+.BR memkind_update_cached_stats ()
+is used to force an update of cached dynamic allocator statistics.
+Statistics are periodically updated by memkind library and this method allows to force update it by demand.
+.PP
+.BR memkind_get_stat ()
```

Comment:
You should mentions that stats can be stale and should be refreshed with `memkind_update_cached_stats()` when (...). (describe when)